modules/network/src/main/java/org/apache/ignite/network/DefaultMessagingService.java

```diff
@@ -275,7 +280,13 @@ private void sendToSelf(NetworkMessage msg, @Nullable Long correlationId) {
      */
     private void onMessage(InNetworkObject obj) {
         if (isInNetworkThread()) {
-            inboundExecutor.execute(() -> onMessage(obj));
+            inboundExecutor.execute(() -> {
+                try {
+                    onMessage(obj);
+                } catch (RuntimeException e) {
+                    LOG.warn("onMessage() failed while processing " + obj.message() + " from " + obj.consistentId(), e);
```

Comment:
Switched it to `{}` style, added a guard to only do this work when the corresponding logging level is enabled. Switched the level to INFO (because we need to see these messages, at least during the active development phase, this really helps in debugging faulty runs on TC).